src/ui/flutter_app/lib/shared/number_picker.dart

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
This component can be used more widely in other configuration items in the future.

Reply:
Yes it does. Is there anything preventing this rn? 

Also keep in mind that the scaling isn't quite right for now. Please remind me if I forget to come back before finishing this pr.

Reply:
What does rn mean? 

This component may not be used in scenes that allow adjustments to the decimal point.

Reply:
>What does rn mean?

right now

> This component may not be used in scenes that allow adjustments to the decimal point.

I don't think a selector with decimal numbers will work. As you would need to specify the divisions between the numbers. In this case a slider should be used

> The user may increase the font size and need to ensure the aesthetics of the large font, which is troublesome.

I don't see what is holding us back